src/teamsnap-ui.scss

```diff
@@ -1,3 +1,5 @@
+$responsive: true;
```

Comment:
Hmmm... let's just set it as the default in the config instead. Rather than setting it as false in the config and overriding for everywhere.

_Follow up :_ we might need to think more about how we turn the responsiveness off and on when we start implementing in the app too.  Just leaving it on is fine for now and people can just use the appropriate classes, but when we get to a point where we want to start using those responsive classes and turn the responsiveness of those classes off and on, just generating or not generating the class won't work.  I think we'll have to add the classes regardless and set the breakpoints values to 0 like we do in classic : https://github.com/teamsnap/classic/blob/master/app/assets/stylesheets/framework/settings/_layout.scss#L34

Reply:
@williamdingwall  I'm not totally following this statement:

> when we get to a point where we want to start using those responsive classes and turn the responsiveness of those classes off and on

Let's discuss soon. It's okay to leave as-is for the moment, though?

Reply:
Yeah that's fine, we can talk monday.. and I'll update in the re-organize story